STYLEGUIDE.md

```
@@ -20,6 +20,7 @@ Regras para escrever, estruturar e manter pipelines neste monorepo. Cada regra d
 10. [`pyproject.toml`](#10-pyprojecttoml)
 11. [`src/prefect_rj_iplanrio/` — código compartilhado do workspace](#11-srcprefect_rj_iplanrio--código-compartilhado-do-workspace)
 12. [Higiene do repositório](#12-higiene-do-repositório)
+13. [Labels de pipeline](#13-labels-de-pipeline)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)